src/lib/coaler/embedder/ConformerEmbedder.cpp

```diff
@@ -47,4 +69,42 @@ namespace coaler::embedder {
         params.numThreads = m_threads;
         RDKit::DGeomHelpers::EmbedMultipleConfs(*mol, numConfs, params);
     }
+
+    /*----------------------------------------------------------------------------------------------------------------*/
+
+    bool ConformerEmbedder::embedEvenlyAcrossAllMatches(RDKit::ROMol& mol, unsigned minNofConfs, unsigned maxNofConfs) {
```

Comment:
is this ready for merge? this method is not called as far as I could see

Reply:
Ihr müsst übrigens mal gucken, wie ihr das mit meinem ansatz vereint. Es gibt leider das problem, dass man manchmal auf einen match auch gar keine conformere generieren kann. Also da liefert der embedder einfach nicht. Das müsste man noch mit bedenken wenn man pro match conformere erstellen will. Ich weiß auch nicht so genau, ob das generell die geilste lösung ist einfach für alle matches was zu erstellen. Das geht vlt bei so rotationssymetrischen dingen. Aber wenn der core an mehreren stellen in einem großen molekül matcht, kriegt man dann den core an stellen wo man ihn vlt. nicht will. müssten wir vlt nochmal zusammen besprechen.

Reply:
for the chunking: I would love to use this for it
https://en.cppreference.com/w/cpp/ranges/chunk_view

although this is a cpp 23 feature. Not sure if we can bump the standard this high

Reply:
I checked it out. If I am not mistaken, the last bucket is the only "underfilled" one. However for our use case, I want an even distribution s.t. for no two buckets the count difference is larger than 1. You can refer to the tests for examples.